### ISSUE TITLE
failing to parse a random file in /sites-available/ prevents certbot apache from renewing certificates

### ISSUE THREAD

**Issue by jsoref** [certbot/certbot#5225]
## My operating system is (include version):
ubuntu 16.04.3 (xenial)

## I installed Certbot with (certbot-auto, OS package manager, pip, etc):
apt

## I ran this command and it produced this output:
`systemctl status certbot.service`
```
Nov 06 12:11:32 hostname certbot[27647]: The error was: PluginError('There has been an error in parsing the file /etc/apache2/sites-available/README on line 1: Syntax error',). Skipping.
Nov 06 12:11:32 hostname certbot[27647]: All renewal attempts failed. The following certs could not be renewed:
Nov 06 12:11:32 hostname certbot[27647]:   /etc/letsencrypt/live/a.example.com/fullchain.pem (failure)
Nov 06 12:11:32 hostname certbot[27647]:   /etc/letsencrypt/live/b.example.com/fullchain.pem (failure)
Nov 06 12:11:32 hostname certbot[27647]:   /etc/letsencrypt/live/c.example.com/fullchain.pem (failure)
Nov 06 12:11:32 hostname certbot[27647]: 3 renew failure(s), 0 parse failure(s)
```

## Certbot's behavior differed from what I expected because:
1. The only files whose parsing should matter is the ones in `/sites-enabled/` not the ones in `/sites-available/`
2. It's a bit odd that there's a "parse failure" category and this error didn't fall into that bucket.

###### Logs are stored in `/var/log/letsencrypt` by default. Feel free to redact domains, e-mail and IP addresses as you see fit.

```
2017-11-06 12:11:32,077:DEBUG:certbot.storage:Should renew, less than 30 days before certificate expiry 2017-12-01 18:14:00 UTC.
2017-11-06 12:11:32,077:INFO:certbot.renewal:Cert is due for renewal, auto-renewing...
2017-11-06 12:11:32,078:DEBUG:certbot.plugins.selection:Requested authenticator apache and installer apache
2017-11-06 12:11:32,249:DEBUG:certbot_apache.configurator:Apache version is 2.4.18
2017-11-06 12:11:32,431:DEBUG:certbot.plugins.disco:Other error:(PluginEntryPoint#apache): There has been an error in parsing the file /etc/apache2/sites-available/README on line 1: Syntax error
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/certbot/plugins/disco.py", line 130, in prepare
    self._initialized.prepare()
  File "/usr/lib/python2.7/dist-packages/certbot_apache/configurator.py", line 197, in prepare
    self.check_parsing_errors("httpd.aug")
  File "/usr/lib/python2.7/dist-packages/certbot_apache/augeas_configurator.py", line 79, in check_parsing_errors
    raise errors.PluginError(msg)
PluginError: There has been an error in parsing the file /etc/apache2/sites-available/README on line 1: Syntax error
2017-11-06 12:11:32,432:DEBUG:certbot.plugins.selection:No candidate plugin
2017-11-06 12:11:32,432:DEBUG:certbot.plugins.selection:No candidate plugin
2017-11-06 12:11:32,432:DEBUG:certbot.plugins.selection:Selected authenticator None and installer None
2017-11-06 12:11:32,432:INFO:certbot.main:Could not choose appropriate plugin: The apache plugin is not working; there may be problems with your existing configuration.
The error was: PluginError('There has been an error in parsing the file /etc/apache2/sites-available/README on line 1: Syntax error',)
```

## Here is the relevant nginx server block or Apache virtualhost for the domain I am configuring:
There's a README file in sites-available. No one will call `a2ensite README`, it's a README. This should not break anything.

**Comment by joohoi** [comment]
This got fixed in #4104 with many related issues, and the first release that includes the fixes would be v0.19, which was released October 5th. 

If you don't have a possibility to upgrade Certbot, the problem can be worked around by passing parameter `--apache-vhost-root=/etc/apache2/sites-enabled/` on the command line.

I'll close the issue as resolved. If the problem persists after upgrade, feel free to comment here and I'll reopen the issue.

**Comment by jsoref** [comment]
Fwiw, I was running 0.17 (0.17.0-2+ubuntu16.04.1+certbot+1).
I "fixed" my problem by prefixing each line in `README` with `#`.
But I'm glad that this shouldn't occur in the future.
